casbin_async_sqlalchemy_adapter/adapter.py

```diff
@@ -95,6 +100,11 @@ async def _session_scope(self):
                 await session.rollback()
                 raise e
 
+    async def create_table(self):
```

Comment:
See other adapters (like Go, Java) about how to create table. This is not Casbin-like

Reply:
I know it's abrupt, and as I mentioned in the question, you can't call asynchronous code directly in python's `__init__()` method, so it's forced.

Unless, of course, there is a better implementation @Wrapping-2000